scripts/estimate_memory_usage.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python3
+
+import argparse
+import math
+
+base_mem_requirement = 32
```

Comment:
Knit: First impression: Why not 42 ? 32 of what ? Megabytes ? `base_mem_gig` would also make sense.

Reply:
32MB is a bit more then an empty usage measured by VisualVM. I renamed to variable to make the units cleaner.